apps/src/code-studio/showProjectAdmin.js

```diff
@@ -39,40 +39,22 @@ export default project => {
     $('#unpublished_warning').show();
   }
 
-  if ($('#feature_project').length && project.isProjectLevel()) {
-    $('#feature_project').click(function () {
-      var url = `/featured_projects/${project.getCurrentId()}/feature`;
+  if ($('#save_project').length && project.isProjectLevel()) {
+    $('#save_project').click(function () {
```

Comment:
Just so I'm totally clear, "save" effectually means "earmark this project as a potential featured project for later review"?

Reply:
My hesitation is the "saved" is kind of overloaded word in software and usually refers to storing something in a database in my opinion. Would it be more clear for future if the verb was something like "shortlist"?

Reply:
> Just so I'm totally clear, "save" effectually means "earmark this project as a potential featured project for later review"?

Yes - that's correct.

> My hesitation is the "saved" is kind of overloaded word in software and usually refers to storing something in a database in my opinion. Would it be more clear for future if the verb was something like "shortlist"?

@samantha-code - lmk what you think about using 'shortlist' or another term instead of 'saved'.

Reply:
Update - we're going to use `bookmarked` instead of `saved`.